release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,21 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-column-management"]
+==== Column management
+
+A *Manage columns* icon image:manage-columns.png[title="Manage Columns icon"] is
+now added to some resources pages. When you click on the icon, default column
+names are listed with check boxes left side of the modal and additional column
```

Comment:
s/are listed with check boxes left side of/are listed with check boxes on the left side of/